### PR TITLE
Add new libunwind8

### PR DESCRIPTION
**Pull request by woky**
Also add it as dependency to dotnet-runtime-6.0.

### CODE REVIEW

File: slices/libunwind-13.yaml [hunk 1]
```diff
@@ -4,5 +4,7 @@ slices:
     essential:
       - libc6_libs
     contents:
-      /usr/lib/llvm-13/lib/libunwind.so.*:
-      /usr/lib/x86_64-linux-gnu/libunwind.so.*:
+      /usr/lib/llvm-13/lib/libunwind.so.1.*:
+      /usr/lib/llvm-13/lib/libunwind.so.1:
```
Comment (niemeyer) [inline slices/libunwind-13.yaml:8]: So libunwind8 ships .so.8 while libunwind-12 and -13 ship .so.1?  Gotta love these conventions.

**Review by niemeyer — APPROVED** [review]
Looks reasonable.

File: slices/libunwind-13.yaml (hunk 1 above)
Comment (woky) [reply slices/libunwind-13.yaml:8]: Yep \o/

Comment (cjdcordeiro) [reply slices/libunwind-13.yaml:8]: :man_facepalming:

**Review by cjdcordeiro — APPROVED** [review]
good catch...weird conventions indeed. lgtm

**Pull request merged**